assets/components/ovn/configmap.yaml

```diff
@@ -0,0 +1,36 @@
+---
+# The ovnconfig config file. Used by both node and master processes.
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: ovnkube-config
+  namespace: openshift-ovn-kubernetes
+data:
+  ovnkube.conf:   |-
+    [default]
+    mtu="1400"
+    cluster-subnets="10.128.0.0/14/23"
+    encap-port="6081"
+    enable-lflow-cache=false
+    lflow-cache-limit-kb=870
+
+    [kubernetes]
+    service-cidrs="172.30.0.0/16"
+    ovn-config-namespace="openshift-ovn-kubernetes"
+    kubeconfig="/var/lib/microshift/resources/kubeadmin/kubeconfig"
```

Comment:
This path is definitely set internally in microshift based on an option that sets the parent directory, so we should have the value substituted before this config is written out.

Reply:
fixed by passing the kubeconfig path param when applying ovnk configmap.